source/Grid/Grid.js

```diff
@@ -381,6 +384,31 @@ export default class Grid extends PureComponent {
     })
   }
 
+  /**
+   * Ensure offset position is visible
+   * Useful for animating position changes
+   */
+  scrollToPosition ({
+    scrollLeft = 0,
+    scrollTop = 0
+  } = {}) {
+    this._setScrollPosition({ scrollLeft, scrollTop })
+  }
+
+  /**
+   * Gets a calculated value to be used for `scrollLeft`
+   */
+  getScrollLeft (props = this.props, state = this.state) {
```

Comment:
They accept props and state mainly because of laziness and not breaking out the parts I think we _actually_ care about. 😅 

 The main reason I see this being used for is computing a "target" `scrollTop` (or `scrollLeft`) based on row or column index. Really, the only things I need to pass are the indexes and the alignment.

```
getOffsetForCell({ columnIndex, rowIndex, scrollToAlignment }) => { scrollLeft, ScrollTop }
```

seems perfectly reasonable.